pkg/manager/internal/controllers/organization_controller.go

```diff
@@ -152,3 +172,93 @@ func (r *OrganizationReconciler) reconcileNamespace(ctx context.Context, log log
 	}
 	return nil
 }
+
+func (r *OrganizationReconciler) reconcileOrganizationRoleTemplates(ctx context.Context, organization *corev1alpha1.Organization) ([]*corev1alpha1.OrganizationRoleTemplate, error) {
```

Comment:
why are you returning the temples when you're dropping the result?

Reply:
Yeah, the reason that I dropped the result for that time point is that `OrganizationRoleTempalte` controller was not implemented, so we don't need to check `OrganizationRoleTemplate` status.
But we should actually check  if `OrganizationRoleTempaltes` are ready, and if they are not ready, organization should also be unready.
I added a simple `OrganizationRoleTempalte` controller to just set it to ready, now I am using the results. PTAL